R/quantity.R

```diff
@@ -102,28 +92,52 @@ Quantity <- R6::R6Class(
     #' Print the object to the console
     #' @param ... Rest arguments.
     print = function(...) {
-      private$.printQuantity()
-      private$.printLine("Quantity Type", self$quantityType)
+      ospsuite.utils::ospPrintClass(self)
+      ospsuite.utils::ospPrintItems(list("Quantity Type" = self$quantityType))
+      ospsuite.utils::ospPrintItems(list("Path" = self$path))
+      ospsuite.utils::ospPrintItems(list("Value" = self$getPrintValue()))
+      ospsuite.utils::ospPrintHeader("Formula", level = 2)
+      self$formula$printFormula()
+      if (!self$isConstant && !self$isDistributed) {
+        ospsuite.utils::ospPrintItems(list("Value overrides formula" = self$isFixedValue))
+      }
     },
     #' @description
     #' Print the name of the quantity and its value
     printValue = function() {
+      lifecycle::deprecate_warn(
+        when = "12.2.0.9006",
+        what = I("ospsuite::Quantity$printValue()")
+      )
       self$printQuantityValue(self$name)
     },
     #' @description
     #' Print the value (in scientific notation with 2 digits when needed) and unit of the quantity
     #' @param  caption Text to prepend to the value
     printQuantityValue = function(caption) {
+      lifecycle::deprecate_warn(
```

Comment:
`Quantity` had 3 public print methods and one private!! Refactored it a bit so only `print` and `getPrintValue` are required. The other two public methods are deprecated, the private method is gone.